src/PluginClient.php

```diff
@@ -79,7 +78,7 @@ public function sendRequest(RequestInterface $request)
         $pluginChain = $this->createPluginChain($this->plugins, function (RequestInterface $request) {
             try {
                 return new FulfilledPromise($this->client->sendRequest($request));
-            } catch (HttplugException $exception) {
+            } catch (\Exception $exception) {
```

Comment:
So: write ducumentation that explains that only HTTPlug exceptions can be thrown in the plugin. And that's it? 

What if a plugin cause a runtime exception? How is that handled? Should the next plugin see it?